contracts/FeeManager/FeeManager.sol

```diff
@@ -7,17 +7,27 @@ import "../PartnerManager/IPartnerManager.sol";
 import "../Registrar/IBaseRegistrar.sol";
 import "../Renewer/IBaseRenewer.sol";
 
-error ZeroBalance();
-error NotAuthorized(address sender);
-error TransferFailed(address from, address to, uint256 amount);
-
 /**
     @author Identity Team @IOVLabs
-    @title FeeManager
-    @dev Keeps the balances of the collected revenew made by the partners.
+    @title  Keeps track of the balances of the collected revenue made by the partners.
 */
 contract FeeManager is IFeeManager {
     RIF private _rif;
+    /**
```

Comment:
maybe we just move these to the interface?

Reply:
agreed